### ISSUE TITLE
aving settings after closing the program...

### ISSUE THREAD

**Issue by mikhailmihalkov** [gerbv/gerbv#149]
After reopening the program, the latest settings are not saved... (View units )/( rendering) All reset to mil and Fast by default

**Comment by ooxi** [comment]
Thanks for the report @mikhailmihalkov! Which version of gerbv are you using on which operating system?

**Comment by mikhailmihalkov** [comment]
> Спасибо за отчет@mikhailmihalkov! Какую версию gerbv вы используете в какой операционной системе?

2.9.4 win 10/64

**Comment by mikhailmihalkov** [comment]
oxxi There is a version for linux
https://launchpad.net/ubuntu/+source/gerbv/2.9.4-1/+build/24626945

**Comment by ooxi** [comment]
It will take a bit to review this on my side

**Comment by ooxi** [comment]
OK I can reproduce the issue. Looking into the source code, it does not seem like this feature ever existed.

@mikhailmihalkov to confirm, is this a regression report or a feature request?

**Comment by mikhailmihalkov** [comment]
Эта функция работает на ubuntu но не работает на win

**Comment by ooxi** [comment]
Thanks! Then I might have missed something, will look into it

**Comment by mikhailmihalkov** [comment]
> Спасибо! Тогда я мог что-то упустить, посмотрю

When installing from the official repository, deb package

**Comment by ooxi** [comment]
OK this might have something to do with issue #144. My current working hypothesis is that the schema cannot be found and thus storing / restoring of setting in Windows is disabled.

**Comment by mikhailmihalkov** [comment]
In my understanding, the settings should be saved in the configuration file so it works on ubuntu.. After closing the program, the latest settings are saved and do not require re-entering during subsequent starts

**Comment by ooxi** [comment]
@mikhailmihalkov yes, this is my understanding, too. And I suspect something going wrong while looking up the schema, thus disabling this functionality.

**Comment by StephaneIFO** [comment]
Hi, 

I can confirm this problem on v2.9.8 under Windows10 H2
While waiting for this function to be fixed, is it possible to locally modify a file so that the default unit is mm and not mils?

thanks
